packages/uniforms-mantine/src/ListItemField.tsx

```diff
@@ -1,20 +1,24 @@
+import { List, Group, Box } from '@mantine/core';
 import React, { ReactNode } from 'react';
 import { connectField } from 'uniforms';
-import { List, Group, Box } from '@mantine/core';
 
 import AutoField from './AutoField';
 import ListDelField from './ListDelField';
 
 export type ListItemFieldProps = { children?: ReactNode; value?: unknown };
 
 function ListItem({
-  children = <AutoField label={null} name="" />,
+  children = <AutoField label={null} name="" style={{ width: '100%' }} />,
```

Comment:
Did you test it with custom children? We don't want to pass `width` manually to every child component.